<launch>
   <arg name="ip" default="192.168.0.70"/>
   <arg name="ws" default="2"/>
   <!-- ip 192.168.0.70   ws 2 -->

   <include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch">
      <arg name="port" value="9090" />
   </include>

   <node pkg="vrx_gazebo" type="connect_ws_jackal_wamv.py" name="connect_ws" output="screen">
      <param name="~ip" value="$(arg ip)" />
      <param name="~ws" value="$(arg ws)" />
   </node>

</launch>
